draft-ietf-rtcweb-jsep.xml

```diff
@@ -2311,7 +2311,7 @@ as specified in <xref target="RFC5245" format="default"/>.</li>
           possible, and media sections will not be marked as bundle-only. This is
           by design, but could cause issues in the rare case of sending a 
           subsequent offer as an initial offer to a non-bundle-aware endpoint
-          via Third Party Call Control (3PCC).</t>
+          via Third Party Call Control (3PCC), see <xref target="I-D.ietf-mmusic-rfc8843-bis" sectionFormat="comma" section="7.6"/>.</t>
```

Comment:
I'd suggest either ", as discussed in <xref..." or the parenthetical "(see <xref..."